### PR TITLE
Adds default security headers in next.config

### PR DESCRIPTION
**Pull request by gustavoguichard**
I thought about leaving Youtube for videos and Google for fonts as a start point, what do you think @diogob ?

### CODE REVIEW

File: ui/next.config.js [hunk 1]
```diff
@@ -2,4 +2,55 @@ const withTM = require('next-transpile-modules')(['domain-logic'])
...
+        headers: securityHeaders,
+      },
+    ]
+  },
 })
+
+// The declaration below will allow Youtube Videos and Google Fonts
+const ContentSecurityPolicy = `
+  default-src 'self';
+  script-src 'self' 'unsafe-eval' 'unsafe-inline' *.youtube.com;
```
Comment (diogob) [inline ui/next.config.js:18]: I'm not sure if we need unsafe-eval e unsafe-inline

File: ui/next.config.js [hunk 2]
```diff
@@ -2,4 +2,55 @@ const withTM = require('next-transpile-modules')(['domain-logic'])
...
+  child-src *.youtube.com;
+  style-src 'self' 'unsafe-inline' *.googleapis.com;
+  img-src * blob: data:;
+  media-src 'none';
+  connect-src *;
```
Comment (diogob) [inline ui/next.config.js:23]: having a * here seems too broad. 
Usually we just need to connect to the main API (which we could fetch from the environment) and a few thrid party providers (which the developer could manually specify).

File: ui/next.config.js [hunk 3]
```diff
@@ -2,4 +2,55 @@ const withTM = require('next-transpile-modules')(['domain-logic'])
...
+    ]
+  },
 })
+
+// The declaration below will allow Youtube Videos and Google Fonts
+const ContentSecurityPolicy = `
+  default-src 'self';
+  script-src 'self' 'unsafe-eval' 'unsafe-inline' *.youtube.com;
+  child-src *.youtube.com;
+  style-src 'self' 'unsafe-inline' *.googleapis.com;
```
Comment (diogob) [inline ui/next.config.js:20]: having unsafe-inline for styles is potentially useless as well

File: ui/next.config.js (hunk 1 above)
Comment (gustavoguichard) [reply ui/next.config.js:18]: It seems that we need them at least for development, not sure about production.
Screenshot 2021-09-09 às 14 29 31

I know Next injects styles in Dev and not in production, but about the scripts we'll probably need more allowance.

I'll play a little bit to find a good balance

**Review by diogob — APPROVED** [review]
LGTM

File: ui/next.config.js (hunk 1 above)
Comment (gustavoguichard) [reply ui/next.config.js]: Now we'll only have `unsafe-eval` in development ;)

File: ui/next.config.js (hunk 2 above)
Comment (gustavoguichard) [reply ui/next.config.js]: Done

File: ui/next.config.js (hunk 3 above)
Comment (gustavoguichard) [reply ui/next.config.js]: Now we'll only have `unsafe-inline` in development

it seems that Nextjs only need those in development mode \o/

**Pull request merged**